mocha.js

```diff
@@ -776,7 +776,7 @@ module.exports = function(suite) {
   });
```

Comment:
Hey @simovm! Thank you for checking this out! 🤓

I agree that this raises a question mark, and this is why I’m bringing this up. I’m not familiar with the process either — this is my first PR — so I guess in this case it’s appropriate to ask those who know. 🤔